src/browserosaurus.json

```diff
@@ -0,0 +1,66 @@
+{
+  "version": "<%= version %>",
+  "settings": {
+    "configMode": "override"
+  },
+  "browsers": [
+    {
+      "name": "Brave",
+      "key": "b"
+    },
+    {
+      "name": "Chromium",
+      "key": "c"
+    },
+    {
+      "name": "Firefox",
+      "key": "f "
```

Comment:
space in here is preventing f shortcut key working

Reply:
TODO: 
- trim() on value
- some kind of UTF8 sanitization 
- take only the first charactet (just in case someone put "ab" or the likes of it)

I have no time to investigate how ```mousetrap``` library handle edge-case input, so i'll assume that in those cases It will just work or not, without breaking anything.
Also I have faith in the user.

Reply:
I think it would be safe to assume that the user will be responsible for setting the proper info, as long as clear documentation is provided.

Reply:
Yes. We can pre-trim. Good shout.